gcsstore/gcsstore.go

```diff
@@ -257,3 +258,7 @@ func (store GCSStore) GetReader(id string) (io.Reader, error) {
 
 	return r, nil
 }
+
+func (store GCSStore) SupportsDeferredLength() bool {
+	return true
```

Comment:
Frankly, I am not so sure whether GCSStore supports this extension out of the box.